### PR TITLE
[CHK-3338] feat: remove old last usage method

### PR DESCRIPTION
**Pull request by scaminati-bv**
#### List of Changes
- remove _pagopa-wallet-usage-update-queue_ connection
- remove queue insertion of _WalletLastUsageEvent_ on _TransactionRequestAuthorization_ handler

#### Motivation and Context
This implementation is not used anymore and is replaced by new last used payment method 

#### How Has This Been Tested?
Unit test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by pietro-tota — DISMISSED** [review]

**Review by ciuffagianluca — APPROVED** [review]

**Review by ciuffagianluca — CHANGES_REQUESTED** [review]
Please resolve conflicts

File: .devops/code-review-pipelines.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-3316-align-env
```
Comment (pietro-tota) [inline .devops/code-review-pipelines.yml:15]:
```suggestion
      ref: main
```

**Review by pietro-tota — DISMISSED** [review]

File: .devops/code-review-pipelines.yml (hunk 1 above)
Comment (ciuffagianluca) [inline .devops/code-review-pipelines.yml:15]:
```suggestion
      ref: main
```

**Review by ciuffagianluca — APPROVED** [review]

File: .devops/code-review-pipelines.yml (hunk 1 above)
Comment (scaminati-bv) [reply .devops/code-review-pipelines.yml]: done

**Pull request merged**